README.md

```diff
@@ -139,3 +140,14 @@ let start = firstCell.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
 let finish = firstCell.coordinateWithNormalizedOffset(CGVectorMake(0, 6))
 start.pressForDuration(0, thenDragToCoordinate: finish)
 ````
+
+### Move cursor position within text field
+
+In this example, a table view cell contains a text field with some content. To move the cursor within the text field, create a `XCUICoordinate` from the left edge of the text field, tap & hold and then drag `dx` points to the right.
+
+````swift
+let textField = cells.textFields.elementBoundByIndex(1)
+let start = textField.coordinateWithNormalizedOffset(CGVectorMake(0, 0))
+let finish = start.coordinateWithOffset(CGVectorMake(70, 0))
```

Comment:
Does 70 *always* work? I'm worried that a lowercase <key>l</key> might take up less space than an uppercase <key>W</key> and start to cause problems. Also, what about international characters?